README.md

```diff
@@ -91,6 +91,33 @@ try &mdash; it does not require you to complete these steps and instead it will
 download and install GatorGrader and run all of the preconfigured checks when
 you type `gradle grade` in your terminal window.
 
+
+### Python and Pipenv Installation
+
+#### Python Installation
+Python comes pre-installed on many different distributions, and is available as
+a package on all linux distributions. However there are certain features you
+might want to use that are not available on your distro’s package. Simply google
+`python download` for your OS, and follow the download and installation 
+instructions on [the Python website](python.org).
+
+#### Pipenv Installation
+[Pipenv](https://github.com/pypa/pipenv) is used by gatorGrader to: create a
+virtual testing environment, install and manage development packages, and to run
```

Comment:
discard the `to` from before `run`; it does not fit with the style of your other list items.